docs/examples/run_coordinate_transform.py

```diff
@@ -1,9 +1,18 @@
+"""
+Coordinate Transform
+====================
+
+This script writes two positions using the high content screening (HCS)
+OME-Zarr dataset with two FOV in a single well with different
+coordinate transformations (i.e translation and scaling)
+"""
+
 # %%
-# This script writes two positions using the high content screening (HCS)
-# OME-Zarr dataset with two FOV in a single well with different
-# coordinate transformations (i.e translation and scaling)
+# Setup
+# -----
```

Comment:
Is there any reason for the `Setup/-----` as a header? This is the only example with this

Reply:
Not really... Will remove for consistency.